src/components/Page/styles.module.scss

```
@@ -32,6 +38,41 @@
+    display: flex;
+    min-width: 0;
+    align-items: center;
+    align-self: stretch;
+    flex: 1 1 auto;
+}
+
+.deckNavRail {
+    position: absolute;
+    z-index: 1;
+    top: 50%;
+    // The header shell already spans the full page width — from the sidebar edge
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)